src/scripts/generate-unary-ops.py

```diff
@@ -0,0 +1,206 @@
+            if op in ["++", "--", "~", "!"]:
+                if op in ["~", "!"]:
+                    operation = self.create_declarations(operation, self.TEST_VALUE_VAR_NAME)
+                operation = ("for (i = 0; i < %d; i++) {\n%s}\n" % (count, operation))
+            elif op in ["+", "-"]:
+                operation = "%s" % (self.create_declarations(operation, self.TEST_VALUE_VAR_NAME))
+        elif op == "typeof":
+            operation = "%s" % (self.create_declarations("%s(%s)" % (op, self.TEST_VALUE_VAR_NAME.format(NUMBER = "")), self.TEST_VALUE_VAR_NAME))
+        elif op == "delete":
+            if i % 2:
+                operation = "%s" % (self.create_declarations("%s %s[%d]" % (op, self.TEST_VALUE_VAR_NAME.format(NUMBER = ""), random.randint(0, count - 1)), self.TEST_VALUE_VAR_NAME))
+            else:
+                operation = "%s" % (self.create_declarations("%s %s" % (op, self.TEST_VALUE_VAR_NAME.format(NUMBER = "")), self.TEST_VALUE_VAR_NAME))
+        elif op == "void":
+            operation = "%s" % (self.create_declarations("typeof(%s %s())" % (op, self.TEST_VALUE_VAR_NAME.format(NUMBER = "")), self.TEST_VALUE_VAR_NAME))
+        return operation
+
+    def append_test_case(self): # Appends test_case to file output
+        self.file_output.append(self.test_case)
+        self.test_case = ""
+
+    def configure_settings(self, options):
+        return settings.file_write_settings(
+            _GENERATED_FILENAME,
+            options.output,
+            validate.validate_numeric,
+            _constants.test_case_in_a_file)
+
+    def add_unique_test_cases(self, op): # Adds Unique test cases for some operators
+        if op in ["+", "-", "typeof"]:
+            if op in ["+", "-"]:
+                for step in ["true","false","null"]:
+                    self.expected_value = step == "true"
+                    if op == "-":
+                        self.expected_value = -self.expected_value
+                    self.expected_value = int(self.expected_value)
+                    false_values = self.generate_false_numbers(
+                        self.expected_value,
+                        _FALSE_VALUES_COUNT,
+                        _constants.min,
+                        _constants.max)
+                    false_values.append(int(step != "true"))
+                    self.test_case += self.create_declarations(step, self.TEST_VALUE_VAR_NAME.format(NUMBER = ""))
+                    self.test_case += self.create_declarations(self.expected_value, self.EXPECTED_VAR_NAME)
+                    self.test_case += self.create_declarations(false_values, self.FALSE_RESULT_VAR_NAME)
+                    self.test_case += self.create_declarations("%s%s" % (op, self.TEST_VALUE_VAR_NAME.format(NUMBER = "")), self.TEST_VALUE_VAR_NAME)
+                    self.test_case += self.create_function_call(self.TEST_VALUE_VAR_NAME.format(NUMBER = ""),
```

Comment:
Please apply [this](https://github.com/szeged/js-test-generators/pull/12#discussion_r239354471) in this file as well.